### DISCUSSION TITLE
en_core_web_md produces incorrect lemma for plural forms

### DISCUSSION THREAD

**Discussion by arpitjjw** [explosion/spaCy#13341] [Help: Model Advice]
```
parser = spacy.load("en_core_web_md")

phrases = ['presentation skills',
'marketing solutions',
'marketing campaigns',
'advertising stratgies',
'campaigning plans',
'business tactics',
'social works',
'markert entries',
'finacial modelings']
for word in phrases:
    out = parser(word)
    for token in out:
        print(token.lemma_)
```
Output - `presentation
skills
marketing
solutions
marketing
campaigns
advertising
stratgies
campaigning
plans
business
tactics
social
works
markert
entries
finacial
modelings`

Why doesn't it remove 's' from plural form? Can I edit the lookup table or some lemma_rules in the spacy or model library? I need to hardcode lemma of some phrases. 
Spacy version - 3.4.4

**Comment by svlandeg** [comment]
Hi! I can't reproduce this with spaCy 3.7.2, could you try upgrading and running this again? On my end, the -s do get stripped, and "entries" for instance is mapped to "entry".

**Reply by arpitjjw** [reply]
Actually I just found out that I disabled tok2vec that is why the lemmatization doesn't work. I disabled it because it increased the parsing time by almost 10 times. I don't need vectors so how can I make my model 'en_core_web_md' parse text faster without disabling tok2vec?

**Reply by svlandeg** [reply]
Ok, it sounds like you've been disabling some components upon loading the pipeline (which wasn't clear from the original code snippet). 

Let us back up a second. `en_core_web_md` has the following configuration:
```
pipeline = ["tok2vec","tagger","parser","senter","attribute_ruler","lemmatizer","ner"]
```
The `tok2vec` layer provides word embeddings and is required by the `parser` and the `tagger`. Excluding the parser will get you the biggest speed gain. You need the `tagger` as well as the `attribute_ruler` for the `lemmatizer` to work correctly though. If you don't need access to named entities, then you can also exclude the `ner`.

So you can do:
```
spacy.load("en_core_web_sm", exclude=["parser","ner"])
```

**Reply by arpitjjw** [reply]
Why does 'en_depent_web_md' parses much faster than 'en_core_web_md' (10x slower after disabling parser and ner). When I disable tok2vec 'en_core_web_md' parsing time is comparable to 'en_depent_web_md' but then lemmatization doesn't work. How can I get similar inference time with the new model?

**Reply by svlandeg** [reply]
Which model are you referring to with `en_depent_web_md`?

**Reply by arpitjjw** [reply]
It a model that was used when spacy was 2.x some version.

**Reply by honnibal** [reply]
Model architectures and libraries have changed a lot in the intervening years. Average speeds are definitely not 10x slower than 2.x, so something's not right.

What you're disabling there (the tok2vec) is actually most of the neural network. To make things run faster spaCy puts the expensive neural network to calculate token representations in one place, and then downstream components share that. So the comparison really isn't fair.

I don't remember an en_depent_web_md, that isn't our regular naming structure. I wonder whether this is an old linear model pipeline? If it is then it would indeed be quite fast.

Your machine might be running with slow matrix multiplication kernels, making the neural network take longer than it should.
